apps/src/levelbuilder/ai-tutor/constants.ts

```diff
@@ -1,13 +1,8 @@
-// Non-Gen AI endpoint constants
-const AI_TUTOR_ENDPOINT = 'ai-tutor';
-const LLM_GUARD_ENDPOINT = 'llm-guard';
+import {AiChatModelIds} from '@cdo/generated-scripts/sharedConstants';
 
-// Gen AI endpoint constants
-const GEN_AI_MISTRAL_BASE = 'gen-ai-mistral-7b-inst-v01';
-const GEN_AI_ARITHMO = 'gen-ai-arithmo2-mistral-7b';
-const GEN_AI_BIOMISTRAL = 'gen-ai-biomistral-7b';
-const GEN_AI_KAREN_CREATIVE = 'gen-ai-karen-creative-mistral-7b';
-const GEN_AI_PIRATE = 'gen-ai-mistral-pirate-7b';
+// Non-Gen AI endpoint constants
+export const AI_TUTOR_ENDPOINT = 'ai-tutor';
+export const LLM_GUARD_ENDPOINT = 'llm-guard';
```

Comment:
🎉 Thank you, @sanchitmalhotra126! I'd logged this as a clean-up item to eventually get to https://codedotorg.atlassian.net/browse/LABS-964 and now I can cross it off the list 😁